dags/transportation/dockless/scooter-stat.py

```diff
@@ -80,7 +80,7 @@ def set_xcom_variables(**kwargs):
             COUNT(trip_id)::FLOAT / COUNT (DISTINCT (device_id)) as avg_rides_per_device,
             COUNT (DISTINCT (device_id)) as num_devices_doing_trips
     FROM v_trips 
-    WHERE DATE_PART(start_time_local, 'day') IS '{yesterday}'
+    WHERE DATE(start_time_local') = '{yesterday}'
```

Comment:
n.b., I took a pass at what this stats query would look like using ibis. I still need to upstream a bugfix, but the overall expression looks like this:
```python
dt = trips[trips.start_time_local.date() == ibis.date('2019-10-05')]
grouped = dt.group_by([dt.provider_name, dt.vehicle_type])
expr = grouped.aggregate([
  dt.trip_distance_miles.mean().name('avg_trip_distance'),
  dt.trip_distance_miles.max().name('max_trip_distance'),
  dt.device_id.cast('string').nunique().name('num_devices_doing_trips'),
  (dt.trip_id.count()/dt.device_id.cast('string').nunique()).name('avg_rides_per_device')
])
df = expr.execute()
```